src/collections/_documentation/workflow/integrations/global-integrations.md

```diff
@@ -899,41 +899,39 @@ Sentry owner or manager permissions are required to install this integration. Sl
 
 1. In Sentry, navigate to Organization Settings > **Integrations**.
 
-1. At the top of the page, you’ll see the available Global Integrations list, which includes Slack. Click **Install** to integrate Sentry with your Slack workspace(s).
+1. Once on the page, you’ll see the available Global Integrations list, which will include Slack. You can use the search bar to filter for Slack, if it doesn't appear at the top. 
+Click on **Slack** to go to the details page for the integration.
 
-    [{% asset slack-all-integrations-page.png %}]({% asset slack-all-integrations-page.png @path %})
+    [{% asset slack-integration-directory.png %}]({% asset slack-integration-directory.png @path %})
 
 1. Click **Add workspace**.
 
-    [{% asset slack-auth-choose-workspace.png %}]({% asset slack-auth-choose-workspace.png @path %})
+    [{% asset slack-overview-details-page.png %}]({% asset slack-overview-details-page.png @path %})
 
-    You can toggle the Slack workspace you want to connect in the dropdown menu in the upper right corner of the authentication window. To set up multiple workspaces, go through this process for each separate workspace.
 
-1. After you've connected to the correct workspace, click **Continue**.
+1. You can toggle the Slack workspace you want to connect in the dropdown menu in the upper right corner of the authentication window. To set up multiple workspaces, go through this process for each separate workspace. Once you've chosen the correct workspace, click **Allow**.
```

Comment:
Suggest: "Toggle to the Slack workspace to which you want to connect using the dropdown menu in the upper right corner of the authentication window, then select **Allow**. Repeat this process if you are connecting to multiple workspaces."